### PR TITLE
Autocomplete hotfix – removed datalist elements and replaced with autocomplete plugin

### PR DESCRIPTION
**Pull request by sharissis-brei**

### CODE REVIEW

File: subprojects/degrees-and-programs/source/static/app/js/degreesAndProgramsApp.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ var chapman = chapman || {};
 				var form = $(this),
 					target = $(event.target);
 
-				if (target.attr('id').includes('keyword')) {
+				if (target.attr('id').indexOf('keyword') !== -1) {
```
Comment (tatwell) [inline subprojects/degrees-and-programs/source/static/app/js/degreesAndProgramsApp.js]: Was `includes` not working? I'd expect it to be ok for the browsers we'd support:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Comment (sharissis-brei) [inline subprojects/degrees-and-programs/source/static/app/js/degreesAndProgramsApp.js]: I'm sorry @tatwell, I must have missed the notification for this comment.

`includes` was producing an error in IE11. To be safe, I converted all instances to `indexOf`.

Comment (tatwell) [inline subprojects/degrees-and-programs/source/static/app/js/degreesAndProgramsApp.js]: Good to know. Thanks.

**Comment by tatwell** [comment]
Sorry. I forgot I have to merge this. Usually the dev making the request does.

**Comment by tatwell** [comment]
I merged this into master.

**Pull request merged**